deepchecks/tabular/checks/data_integrity/columns_info.py

```diff
@@ -42,7 +42,7 @@ def run_logic(self, context: Context, dataset_kind) -> CheckResult:
         """
         dataset = context.get_data_by_kind(dataset_kind)
         columns_info = dataset.columns_info
-        columns_info = column_importance_sorter_dict(columns_info, dataset, context.feature_importance)
+        columns_info = column_importance_sorter_dict(columns_info, dataset, context.feature_importance, self.n_top_columns)
```

Comment:
In that case, we can delete lines 47-51, as they are there only to cap the number of displayed columns at self.n_top_columns

Reply:
@noamzbr could you verify the changes?

Reply:
looks good!